src/braket/aws/aws_device.py

```diff
@@ -70,7 +70,7 @@ class AwsDevice(Device):
 
     _GET_DEVICES_ORDER_BY_KEYS = frozenset({"arn", "name", "type", "provider_name", "status"})
 
-    _RIGETTI_GATES_TO_BRAKET = {
+    _RIGETTI_GATES_TO_BRAKET: ClassVar = {
```

Comment:
Is this the correct type?

Reply:
We can define this further in subsequent PRs but this is more explicitly defined as  `ClassVar[dict[str, str]]`